ui-speedspacechart/src/types/chartTypes.ts

```diff
@@ -29,17 +29,34 @@ export type LayerData<T> = {
   value: T;
 };
 
+export type ValuesAlongPath<T> = {
```

Comment:
can you add a TODO to precise that types must be updated to follow this model?

Reply:
And why not start to use it on to new type we add in `Simulation` (mrsp) ?

Reply:
I added a comment above the `LayerData` type.

Reply:
> And why not start to use it on to new type we add in `Simulation` (mrsp) ?

What do you mean? The `Simulation` struct is close from what osrd backend provides so it's easier to test it.

Reply:
I meant in `simulationTypes.ts`, we could use this new type at least for `mrsp` like 
`mrsp: ValuesAlongPath<number>;` since you just added it. But it could also be done for your TODO.